tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react-jsx"
```

Comment:
Mergujemy jednak z tym?

Reply:
😱 oczywiście że nie, już to zmieniam

Reply:
To się chyba teraz automatycznie zmienia przy `npm start`